components/win-users/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "habitat_win_users"
+version = "0.0.0"
+authors = ["Matt Wrock <matt@mattwrock.com>"]
+description = "Habitat library for win32 account api calls"
+build = "../build.rs"
```

Comment:
You only need to specify a build script if you need to compile additional code side by side or control your linking to an external library.  That said, we could pull the C that we have in core over to this crate (as well as the build script), since the checking of UAC/admin status seems to fit feature set for this crate.